<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sorting algorithm demonstration system</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="bubble_btn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(85, 170, 127);
	border-color: rgb(74, 150, 111);
}</string>
      </property>
      <property name="text">
       <string>Bubble Sort</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selection_btn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	background-color: rgb(255, 170, 127);
	border-color: rgb(227, 151, 113);

}</string>
      </property>
      <property name="text">
       <string>Selection Sort</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="insertion_btn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	background-color: rgb(85, 255, 127);
	border-color: rgb(70, 211, 103);
}</string>
      </property>
      <property name="text">
       <string>Insertion Sort</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="merge_btn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	background-color: rgb(255, 255, 127);
	border-color: rgb(227, 227, 113);
}</string>
      </property>
      <property name="text">
       <string>Merge Sort</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quick_btn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	background-color: rgb(85, 170, 255);
	border-color: rgb(76, 152, 228);
}</string>
      </property>
      <property name="text">
       <string>Quick Sort</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="heap_btn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	background-color: rgb(255, 170, 255);
       
	border-color: rgb(233, 155, 233);
}</string>
      </property>
      <property name="text">
       <string>Heap Sort</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="radix_btn">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	background-color: rgb(85, 255, 255);
	border-color: rgb(75, 227, 227);
}</string>
      </property>
      <property name="text">
       <string>Radix Sort</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>109</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="random_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>590</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>590</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>590</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="play_btn">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Play/Pause</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="next_btn">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="restart_btn">
      <property name="text">
       <string>Restart</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="num_text">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>520</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="array_text">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>550</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>470</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
